Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+                proxy=proxy,
+                client_id=client_id,
+                client_secret=client_secret,
+                callback_url=callback_url,
+                server_url=server_url)
+        else:
+            # Configure JiraOnPremClient
+            # urljoin(url, '/')
+            client = JiraOnPremClient(
+                verify=verify_certificate,
+                proxy=proxy,
+                client_id=client_id,
+                client_secret=client_secret,
+                callback_url=callback_url,
+                server_url=server_url)
+            # client.oauth_start()
+        demisto.debug(f'The configured Jira client is: {type(client)}')
+        # else:
+        #     raise DemistoException('Cloud ID and Server URL cannot be configured at the same time')
+
+        if command == 'test-module':
+            return_results(test_module())
+        elif command in commands:
+            return_results(commands[command](client, args))
+        elif command == 'fetch-incidents':
+            demisto.incidents(fetch_incidents(
+                client=client,
+                issue_field_to_fetch_from=issue_field_to_fetch_from,
+                fetch_query=fetch_query,
+                id_offset=arg_to_number(id_offset) or 0,
+                fetch_attachments=fetch_attachments,
+                fetch_comments=fetch_comments,
+                max_fetch_incidents=arg_to_number(max_fetch) or DEFAULT_FETCH_LIMIT,
+                first_fetch_interval=first_fetch_interval,
+                mirror_direction=mirror_direction,
+                comment_tag_to_jira=comment_tag_to_jira,
+                comment_tag_from_jira=comment_tag_from_jira,
+                attachment_tag_to_jira=attachment_tag_to_jira,
+                attachment_tag_from_jira=attachment_tag_from_jira
+            ),
+            )
+
+        elif demisto.command() == 'get-remote-data':
+            return_results(get_remote_data_command(client=client, args=args, comment_tag_from_jira=comment_tag_from_jira,
+                                                   attachment_tag_from_jira=attachment_tag_from_jira,
+                                                   mirror_resolved_issue=mirror_resolved_issue))
+        elif demisto.command() == 'get-modified-remote-data':
+            return_results(get_modified_remote_data_command(client=client, args=args))
+        elif demisto.command() == 'get-mapping-fields':
+            return_results(get_mapping_fields_command(client=client))
+        elif demisto.command() == 'update-remote-system':
+            return_results(update_remote_system_command(client=client, args=args, comment_tag_to_jira=comment_tag_to_jira,
+                                                        attachment_tag_to_jira=attachment_tag_to_jira))
+        else:
+            raise NotImplementedError(f'{command} command is not implemented.')
+
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
```

Comment:
This row is unnecessary. The traceback is printed as part of the return error func. 
```suggestion
```